apps/monarch/monarch-frontend/src/app/AdminPage.tsx

```diff
@@ -44,4 +44,4 @@ function AdminPage () {
     )
 }
 
-export default withAuthenticator(AdminPage);
+export default withAuthenticator(AdminPage, { hideSignUp: true });
```

Comment:
This setting removes access to the "Sign up" page, only allowing users to log into existing accounts